app/schema/lesson.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+from pydantic import BaseModel
+
+from app.config import get_settings
+from .user import Coach
+from .location import Location
+from .sport import SportTypeSchema
+
+settings = get_settings()
+
+
+class Lesson(BaseModel):
+    name: str
+    location: Location
+    sport: SportTypeSchema
```

Comment:
Broken naming convention between different schemas. Either use Schema suffix everywhere, or don't use it at all (preferable as for me)